### ISSUE TITLE
[Discussion] AI: Context-based steering

### ISSUE THREAD

**Issue by cbscribe** [kidscancode/godot_recipes#75]
Discussion for http://godotrecipes.com/ai/context_map/

**Comment by Zireael07** [comment]
YouTube version says the text version AND sample project are at this link. Unfortunately, this is only half true, there is no sample project.

**Comment by horsegorilla** [comment]
Hello, With this example I placed objects and enemies on a map without a tilemap and gave the enemies a pathfinder. But there is one problem. Since the value is proportional to the length of the position, if there is no object, I made it go in that direction.

![1](http://drive.google.com/uc?export=view&id=1xehOK1IsbEArGEJHx-EANF1WaDwCaRWx)

In this case, the enemies just move in place (if the player is still). In my humble opinion, it seems to repeat itself in between loading values every delta. Is there any way?

Thank you, and sorry for my awful english.

**Comment by agapelogos4** [comment]
Yes a link to the project would be great. Personally I'm having a problem with the "agent" flying to the side of the screen, and not following the path.
https://streamable.com/15ibni
/\ Video example.

**Comment by agapelogos4** [comment]
I fixed the agent pathing bug. you can download **here \/** a working version of the project.
[AI test.zip](https://github.com/kidscancode/godot_recipes/files/7678876/AI.test.zip)
